src/index.tsx

```diff
@@ -436,50 +463,62 @@ class App extends React.Component<{}, AppState> {
   }
 
   public render() {
+    const hasSavedItems = !!localStorage.getItem(LOCAL_STORAGE_KEY);
+
     return (
       <>
-        <div className="exportWrapper">
-          <button
-            onClick={() => {
-              exportAsPNG({
-                exportBackground: this.state.exportBackground,
-                exportVisibleOnly: this.state.exportVisibleOnly,
-                exportPadding: this.state.exportPadding
-              });
-            }}
-          >
-            Export to png
-          </button>
-          <label>
-            <input
-              type="checkbox"
-              checked={this.state.exportBackground}
-              onChange={e => {
-                this.setState({ exportBackground: e.target.checked });
+        <div className="wrappers">
+          <div className="saveWrapper">
+            <button disabled={elements.length === 0} onClick={save}>
+              Save
+            </button>
+            <button disabled={!hasSavedItems} onClick={restore}>
+              Restore
+            </button>
+          </div>
```

Comment:
We should do it automatically whenever a change is done. And restore on page load.